docs/atlascli/command/atlas-deployments-search-indexes-delete.txt

```diff
@@ -0,0 +1,95 @@
+.. _atlas-deployments-search-indexes-delete:
+
+=======================================
+atlas deployments search indexes delete
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Delete the specified search index from the specified cluster.
```

Comment:
In the docs, we've replaced cluster with deployments throughout.

```suggestion
Delete the specified search index from the specified deployment.
```

Reply:
done